src/Csv/FormatInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
@Argee88 This looks like you added the two new files instead of adding the blank lines. What happened here?

Reply:
Possibly you'll need to rebase your branch from master first, resolve any conflicts and then force-repush.

Reply:
Rebased and pushed again... I hope it works this time :/